stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
@dra27: either "follow" or "are as follows" is an improvement, but I'd prefer something a bit less non-committal:

> [backtrace_slots rv] returns [None] in the following circumstances:
>  - every slot in [rv] refers to a module compiled without debug information enabled ([-g])
>  - the program is a bytecode program that has been linked without debug information enabled ([ocamlc -g])